src/main/java/seedu/address/model/util/InternshipItemUtil.java

```diff
@@ -0,0 +1,13 @@
+package seedu.address.model.util;
+
+/**
+ * InternshipItemUtil class which provides the strings used in the InternshipItem.
```

Comment:
Maybe can just remove the InternshipItemUtil class which and just be Provides the string used in ... ? or perhaps A utility class that provides ...

Reply:
Yup, fixed in my latest merge.